site/docs/fargate-tutorials/sql.md

```diff
@@ -79,86 +79,41 @@ VALUES ('quickstart', 8088, true, 'latest');
 SELECT * from iasql_apply();
 ```
 
-<!--- https://www.urlencoder.org/ -->
-<button
-  className={"button button--primary button--lg margin-bottom--lg"}
-  onClick={() => window.open('https://app.iasql.com/#/button/INSERT%20INTO%20ecs_simplified%20%28app_name%2C%20app_port%2C%20public_ip%2C%20image_tag%29%0AVALUES%20%28%27quickstart%27%2C%208088%2C%20true%2C%20%27latest%27%29%3B%0A%0ASELECT%20%2A%20from%20iasql_apply%28%29%3B', '_blank')}
->
-Run SQL
-</button>
-
 If the function call is successful, it will return a virtual table with a record for each cloud resource that has been created, deleted, or updated.
 Login, build and push your code to the container registry
 
-1. Grab your new `ECR URI` from the hosted DB
+Previously, you needed to manually build and push your image to the ECR. But recently we've added the high-level `ecr_build` SQL function which does all those steps automatically. It will do the following:
+- Pull the codes from your Github repository
```

Comment:
Same Nit. `code` is one of those weird words in English that has still has a remnant rule on pluralization, like `fish`.

| Pluralization | `code` | `fish` |
| ------------- | ------ | ------ |
| Singular      | `code` | `fish` |
| Plural of same type | `code` | `fish` |
| Plural of multiple types | `codes` | `fishes` |

Making this:

```suggestion
- Pull the code from your Github repository
```

Because it's all code from the same repo.